server/routes/appointments/appointmentsController.ts

```diff
@@ -382,28 +382,30 @@ export default class AppointmentsController {
     if (appointmentScheduleDetails === null || !appointmentScheduleDetails.appointmentTime) {
       throw new Error('No scheduling details were filled in on the draft')
     }
+
     if (this.isAPastAppointment(appointmentScheduleDetails)) {
       res.redirect(
         `/service-provider/action-plan/${actionPlanId}/appointment/${sessionNumber}/post-session-feedback/edit/${draft.id}/attendance`
       )
-    } else {
-      const success = await this.updateSessionAppointmentAndCheckForConflicts(
-        accessToken,
-        actionPlanId,
-        sessionNumber,
-        appointmentScheduleDetails
-      )
+      return
```

Comment:
this diff is confusing due to some alignment changes. the actual change here is the addition of this return statement, and the removal of the subsequent 'else' clause (which is no longer needed after the return)

Reply:
`?w=1` to the rescue 🙌 

<img width="795" alt="image" src="https://user-images.githubusercontent.com/1526295/149498011-d252a524-89de-437e-ae52-a90dc5e16541.png">

Reply:
cool!

i removed the actual functional change by messing around with this - please reload and see the additional return later in this file.